modules/docs/lib/widgetUtils.tsx

```diff
@@ -0,0 +1,299 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {createComponent, StyledType} from '@workday/canvas-kit-react/common';
+import {useDialogModel, Dialog} from '@workday/canvas-kit-react/dialog';
+import {CanvasColor, colors, type} from '@workday/canvas-kit-react/tokens';
+
+import {MDX, MdxJSToJSX} from './MDXElements';
+import {Hyperlink} from '@workday/canvas-kit-react/button';
+import {docs} from './docs';
+import {Value} from './Value';
+import * as types from '../docgen/docTypes';
+import {fontSizes} from '@workday/canvas-kit-react/tokens/lib/type/fontSizes';
```

Comment:
fine if you keep it this way... but was this necessary or can bit be just `/tokens`?

Reply:
This was an autoimport. I should be importing this instead:

```tsx
import {canvas} from '@workday/canvas-kit-react/tokens';

canvas.type.properties.fontSizes
```

Reply:
Actually, I'm already importing `types`. I can access `fontSizes` from there